src/main/java/com/zendesk/maxwell/bootstrap/MaxwellBootstrap.java

```diff
@@ -0,0 +1,155 @@
+package com.zendesk.maxwell.bootstrap;
```

Comment:
It looks like this file doesn't actually DO the bootstrap, but it inserts a row into maxwell.bootstrap so that Maxwell can do the bootstrap? And it also lets you monitor the progress of the bootstrap?

From the filename, I thought that this command was actually performing a bootstrap. Maybe the file should be named MaxwellBootstrapAdmin or MaxwellBootstrapController or something?